### DISCUSSION TITLE
what if an organization not able to differentiate TAX ID is SSN or not.

### DISCUSSION THREAD

**Discussion by rajkumarsowmy** [CMSgov/price-transparency-guide#288] [Q&A] [answered]
@shaselton-usds
we have a situation where we cant identify TAX_ID is SSN or Not an SSN.
can we make the Value attribute as array as well, so we dont have to repeat this for each npi type?
please let us know your thoughts.

"provider_groups": [{
"npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555], _-- ARRAY TYPE_
"tin":{
"type": "npi",
"value" "22-2222222" -- _currently it is string type, can we make it as array type , so we dont have to repeat this for each npi._
}
}]

**Comment by Vcat1011** [comment]
Maybe if possible,  the field is always masked. So whatever the valud, whether SSN or TIN or ?, it's protected in some way

**Reply by rajkumarsowmy** [reply]
Thank you, but Masking is not a recommended way as per Scott from Last CMS meeting. 
we need to see how BCBSA Association is going to handle this situation.

**Comment by jltang-kp** [comment]
So from how I understand the CMS direction on SSN is if it's known the provider is using their SSN as TIN. I would not think it is a problem if a TIN that is an SSN is mistakenly reported if there is no way of knowing beforehand. IE source system does not have an SSN/TIN indicator.

**Reply by rajkumarsowmy** [reply]
@shaselton @shaselton-usds 
Could you chime in and help in this situation, 
when an organization don't have the capability in the system to differentiate the provider TIN is ssn or not. 

if we choose to provide only the NPI in the layout for the TIN. the file size will be huge. 
can you think through and make the "value" as array?

current provision from cms layout: (type could be ein(when TIN is not SSN) or npi(when TIN is SSN)). 

"provider_groups": [{
"npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
"tin":{
"type": "ein",
"value": "11-1111111"
}
}
]

since the value col is not an array, we are forced to create like below. 

"provider_groups": [{
"npi": [1111111111],
"tin":{
"type": "npi",
"value": "1111111111"
}
},{
"npi": [2222222222],
"tin":{
"type": "npi",
"value": "2222222222"
}
},{
"npi": [3333333333],
"tin":{
"type": "npi",
"value": "3333333333"
}
},{
"npi": [4444444444],
"tin":{
"type": "npi",
"value": "4444444444"
}
},{
"npi": [5555555555],
"tin":{
"type": "npi",
"value": "5555555555"
}
}
}]

**Reply by rajkumarsowmy** [reply]
@shaselton-usds could you please chime in provide your suggestions we need this Value column to be an array so we can put all the NPI in single array instead of repeating the TIN segment 100-200 times in some cases.

**Reply by shaselton-usds** [reply]
How would you match what TIN applies to what NPI if you have a many to many across two arrays? The TIN is to remain an object for the time being.

**Reply by rajkumarsowmy** [reply]
@shaselton @shaselton-usds
Hi Scott, 
if the type : NPI , you have to assume that company is reporting only NPI not the TIN. 
it make sense, to have value as array when Type: NPI. 

my question is when I cant identify the EIN as SSN , I'm left with only once choice of reporting NPI.  so  type : npi and report the same NPIs under TIN block for (type: npi , value : <npi>) 
would it make sense?

**Comment by shaselton-usds** [comment, marked as answer]
@rajkumarsowmy changing the TIN from a single object to an array would remove the ability to match which TINs are to what NPIs for the specific negotiated rate. The TIN is going to remain an object for the time being.

**Reply by rajkumarsowmy** [reply]
@shaselton @shaselton-usds
Hi Scott, 
My situation is I have TIN in my internal DB, grouped all the providers, but I cant identify if its SSN or not. so I'm reporting only NPI. 

yes, it will make sense, when reporting type : ein and value : <ein>( you can identify TINs to NPI)  , but not the 
type : npi and value : <npi>. 

so Updating the value to array object would helps folks when they cant identify the EIN as SSN, they have an opportunity to report only NPIs. 
pls let me know. Thank you for responding to the question. I have been waiting.

**Reply by shaselton-usds** [reply]
@rajkumarsowmy Trying to wrap up some older threads -- it is my understanding that a SSN is an acceptable TIN for a sole proprietorship, thus I'm not sure why there would be multiple providers grouped to a TIN that was someone's SSN.

The TIN will be kept as is to avoid a loss of context of specific NPI/TIN combinations.
